doc/code_style/mom.cpp

```diff
@@ -110,8 +111,7 @@ class HalfLife3 : public ValveGame {
 			// and then indent with a tab:
 			team.relocate(
 				game_engine->get_team()->get_current_hq_location(),
-				gaben.get_table_location()
-			);
```

Comment:
) on the newline should be ok

Reply:
ugh I should revert this file since I only wanted to change the include instructions, but I guess clang-format had other ideas :D